packages/gatsby-plugin-benchmarks/gatsby-node.js

```diff
@@ -0,0 +1,124 @@
+const nodeFetch = require(`node-fetch`)
+
+const { performance } = require(`perf_hooks`)
+
+const bootstrapTime = performance.now()
+
+let apiServer
+let apiPath
+
+class BenchMeta {
+  constructor() {
+    this.flushing = false // Started flushing?
+    this.flushed = false // Completed flushing?
+    this.crashed = false
+    this.localTime = new Date().toJSON()
```

Comment:
I keep on looking at the toJSON() wondering if that's necessary

Reply:
Hmmm easiest to remember. But I [checked the spec](https://tc39.es/ecma262/#sec-date.prototype.tojson) and it's basically calling `.toISOString()` so that's what it should just do directly :)

```
> new Date().toJSON()
'2020-01-15T09:40:35.240Z'
> new Date().toString()
'Wed Jan 15 2020 10:41:02 GMT+0100 (CET)'
> new Date()
2020-01-15T09:41:21.884Z
> String(new Date())
'Wed Jan 15 2020 10:41:31 GMT+0100 (CET)'
> ''+new Date
'Wed Jan 15 2020 10:41:37 GMT+0100 (CET)'
> new Date().toDateString()
'Wed Jan 15 2020'
> new Date().toGMTString()
'Wed, 15 Jan 2020 09:42:13 GMT'
> new Date().toISOString()
'2020-01-15T09:42:20.174Z'
> new Date().toLocaleDateString()
'2020-1-15'
> new Date().toLocaleString()
'2020-1-15 10:42:45'
> new Date().toLocaleTimeString()
'10:42:50'
> new Date().toSource()
TypeError: (intermediate value).toSource is not a function
> new Date().toUTCString()
'Wed, 15 Jan 2020 09:43:08 GMT'
> new Date().valueOf()
1579081397100
```